coriolis/osmorphing/suse.py

```diff
@@ -148,8 +149,13 @@ def _add_cloud_tools_repo(self):
                 " for more details." % repo) from err
 
     def install_packages(self, package_names):
-        self._exec_cmd_chroot(
-            'zypper --non-interactive install %s' % " ".join(package_names))
+        try:
+            self._exec_cmd_chroot(
+                'zypper --non-interactive install %s' % " ".join(package_names)
```

Comment:
Please add the same error message wrapper mentioned above for apt-based systems.